app/code/Ludmila/LSNewClasses/Block/Homework11/Homework_OOP.php

```diff
@@ -0,0 +1,73 @@
+<?php
+namespace Ludmila\LSNewClasses\Block\Homework11;
+use Ludmila\LSNewClasses\Model\ShowFiles;
+use Ludmila\LSNewClasses\Model\ShowConstants;
+use Ludmila\LSNewClasses\Model\ShowTypes;
+
+/**
+ * Class HomeworkOOP
+ * @package Ludmila\LSNewClasses\Block\Homework11
+ */
+class HomeworkOOP extends \Magento\Framework\View\Element\Template {
+    /**
+     * @var ShowFiles
+     */
+    public $showFilesGet;
```

Comment:
I don't like the names of these properties $showFilesGet.  Why they are public? It's unclear what is inside. Why it has 'get' ending? If it is not a method, it shouldn't contain any verbs. 
It it consists the class for getting the files, probably you can call the variable for it filesRetriever, or filesSource, or filesProvider. 
The same is for other properties of this class.